tasks/main.yml

```diff
@@ -1,9 +1,8 @@
 ---
 
 - name: Install required packages
-  dnf:
-    name: dconf-editor
-    state: present
+  action: >
+    {{ ansible_pkg_mgr }} name=dconf-editor state=present
```

Comment:
What are your thoughts on using the `package` module instead? It's cleaner in my opinion and solves the same problem.

https://docs.ansible.com/ansible/latest/modules/package_module.html